boot/bootdata/packages/reactos.dff.in

```diff
@@ -70,6 +70,10 @@ Signature = "$ReactOS$"
 51 = system32\CatRoot
 52 = system32\CatRoot2
 53 = AppPatch
+54 = winsxs
+55 = winsxs\manifests
+56 = winsxs\x86_microsoft.windows.common-controls_6595b64144ccf1df_5.82.2600.2982_none_deadbeef
+57 = winsxs\x86_microsoft.windows.common-controls_6595b64144ccf1df_6.0.2600.2982_none_deadbeef
```

Comment:
@learn-more A quick spot-check on a Windows 10 system I have access to reveals that the equivalent manifest files are named as such:

```
amd64_microsoft.windows.common-controls_6595b64144ccf1df_5.82.10586.0_none_396e892957c7fb25.manifest
amd64_microsoft.windows.common-controls_6595b64144ccf1df_6.0.10586.0_none_8c15ae12515e1c22.manifest
amd64_microsoft.windows.common-controls_6595b64144ccf1df_6.0.10586.839_none_a2ddba30a539a2ae.manifest
x86_microsoft.windows.common-controls_6595b64144ccf1df_5.82.10586.0_none_811bc0006c44242b.manifest
x86_microsoft.windows.common-controls_6595b64144ccf1df_6.0.10586.0_none_d3c2e4e965da4528.manifest
x86_microsoft.windows.common-controls_6595b64144ccf1df_6.0.10586.839_none_ea8af107b9b5cbb4.manifest
```

Thus, the first major and minor parts of the version numbers always stay the same, yet the build and revision parts can and do change to match the build and revision numbers of the version of Windows they were shipped in. (In this case, the computer is running the November update of Windows 10, build 10586. The files ending in `10586.0` are the original copies of these assemblies that were shipped in that release; the files ending in `10586.839` are patched versions delivered via Windows Update.)

That said, this should not cause _that_ much of a problem, because the SxS manifest code I use in my apps lists the version as `6.0.0.0`. Windows then looks for and loads a version that is _at least_ `6.0.0.0`, even if the version found is not _exactly_ `6.0.0.0`. (Of course, which one of the two listed above it actually chooses -- `10586.0` or `10586.839` -- is open to speculation.) Hope this helps!

Reply:
@learn-more : Every time ms changes the way the hash is computed (the last part of the name), the file names should change as well.

Reply:
This DLL shouldn't be installed as a regular file.
Windows has links to avoid using hundreds of identical files, and we already support them.

Reply:
@spetreolle Afaik FAT32 doesn't support links (and ReactOS can't be installed on anything else than FAT32)